app/src/main/java/org/fossasia/openevent/general/search/SearchFragment.kt

```diff
@@ -127,11 +128,22 @@ class SearchFragment : Fragment() {
         rootView.fabSearch.setOnClickListener {
             queryListener.onQueryTextSubmit(searchView.query.toString())
         }
+
+        if (searchViewModel.isQuerying) {
+            Handler().post {
```

Comment:
Why

Reply:
This is a memory leak btw

Reply:
@addiegupta thanks for the warning, I will take a look at that. Initially I user this because there were some problems on rendering the view of the action bar. Will fix this

Reply:
@anhanh11001 😂😂 Wrong Aditya